docker/scala_api/conf/messages

```diff
@@ -0,0 +1 @@
+# https://www.playframework.com/documentation/latest/ScalaI18N
```

Comment:
¿Que función cumple este archivo?

Reply:
Según parece lo podríamos quitar. Según entendí permite hacer entre otras cosas, una especie de variables globales para el lenguaje:
https://objectcomputing.com/resources/publications/sett/january-2014-scala-and-the-play-framework
Estuve probando y no afecta el no tenerlo, pero si uds detectan algo, avisen. Lo eliminé.

Reply:
Probado con los nuevos cambios y sigue funcionando como antes.